src/crab/ebpf_domain.cpp

```diff
@@ -346,13 +348,15 @@ void ebpf_domain_t::scratch_caller_saved_registers() {
 void ebpf_domain_t::forget_packet_pointers() {
     using namespace crab::dsl_syntax;
 
-    initialize_packet(*this);
-
-    for (variable_t v : variable_t::get_type_variables()) {
-        // TODO: this is sufficient, but for clarity it may be useful to forget the offset and value too.
-        if (m_inv.intersect(v == T_PACKET))
-            m_inv -= v;
+    for (variable_t type_variable : variable_t::get_type_variables()) {
+        if (type_inv.has_type(m_inv, type_variable, T_PACKET)) {
+            havoc(variable_t::kind_var(data_kind_t::types, type_variable));
+            havoc(variable_t::kind_var(data_kind_t::packet_offsets, type_variable));
+            havoc(variable_t::kind_var(data_kind_t::values, type_variable));
+        }
     }
+
+    initialize_packet(*this);
```

Comment:
I can't remember since this PR was started long ago.  It does the TODO in the old code.  Let me know if you want it reverted, but since the TODO was there I think it's useful.

Reply:
No need to revert